bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -342,18 +342,40 @@ async def end_nomination_command(self, ctx: Context, user: MemberOrUser, *, reas
             await ctx.send(":x: The specified user does not have an active nomination")
 
     @nomination_group.group(name='edit', aliases=('e',), invoke_without_command=True)
-    @has_any_role(*MODERATION_ROLES)
-    async def nomination_edit_group(self, ctx: Context) -> None:
-        """Commands to edit nominations."""
-        await ctx.send_help(ctx.command)
+    @has_any_role(*STAFF_ROLES)
+    async def nomination_edit_group(self, ctx: Context, target: Member, *, reason: str) -> None:
+        """
+        Commands to edit nominations.
+
+        Edits the reason of the authors nominee in a specific target's active nomination.
+        """
+        await self._edit_nomination_reason(ctx, target=target, actor=ctx.author, reason=reason)
 
     @nomination_edit_group.command(name='reason')
     @has_any_role(*MODERATION_ROLES)
     async def edit_reason_command(self, ctx: Context, nomination_id: int, actor: MemberOrUser, *, reason: str) -> None:
         """Edits the reason of a specific nominator in a specific active nomination."""
+        await self._edit_nomination_reason(ctx, target=nomination_id, actor=actor, reason=reason)
+
+    async def _edit_nomination_reason(
+        self,
+        ctx: Context,
+        target: Union[int, Member],
```

Comment:
```suggestion
        ctx: Context,
        *,
        target: Union[int, Member],
```
Might as well make it kwarg-only if that is how you use it?